eureka-client-jersey2/src/main/java/com/netflix/discovery/shared/transport/jersey2/Jersey2ApplicationClientFactory.java

```diff
@@ -78,14 +80,19 @@ public void shutdown() {
     public static Jersey2ApplicationClientFactory create(EurekaClientConfig clientConfig,
             Collection<ClientRequestFilter> additionalFilters,
             InstanceInfo myInstanceInfo,
-            AbstractEurekaIdentity clientIdentity) {
```

Comment:
I know this is very unlikely to be used outside of internal classes, but give that the method has a public signature, do you mind creating a new overload with the optionals and delegate over (rather than changing) from the original? Thanks.